rhel/SPECS/victoriametrics.spec

```
@@ -54,6 +54,9 @@ install -D -p -m 0755 ./bin/vmalert-pure %{buildroot}%{_sbindir}/vmalert
 
 
 %changelog
+* Thu Dec 24 2020 Vadim Yalovets <vadim.yalovets@percona.com> - 1.50.2-1
+- PMM-7135 Wire everything together
```

upgrade victoriametrics to 1.50.2 release
```suggestion
- upgrade victoriametrics to 1.50.2 release
```